src/main/java/br/com/fiap/techchallenge/infrastructure/controllers/ClienteController.java

```diff
@@ -17,22 +18,32 @@
 @AllArgsConstructor
 @RestController
 @RequestMapping("/api/clientes")
+
 public class ClienteController {
 
-    private ClienteService clienteService;
+    // private ClienteService clienteService;
```

Comment:
```suggestion
    // private ClienteService clienteService;
```

Acho que podemos remover